python/lsst/ts/xml/data/sal_interfaces/GenericCamera/GenericCamera_Commands.xml

```diff
@@ -159,4 +159,21 @@
       <Count>1</Count>
     </item>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>GenericCamera</Subsystem>
+    <EFDB_Topic>GenericCamera_command_startStreamingMode</EFDB_Topic>
+    <Description>Start streaming mode operation of the camera</Description>
```

Comment:
Nitpicking but please terminate all descriptions in a `.`, also in `GenericCamera_Events.xml`.

Reply:
Fixed all of the descriptions.